optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/pages/spotroster/SpotRosterPage.java

```diff
@@ -102,7 +103,7 @@
 
     @Inject
     @DataField("previous-page-button")
-    private HTMLAnchorElement previousPageButton;
+    private HTMLAnchorElement cuurentSpotRosterViewpreviousPageButton;
```

Comment:
Will fix (is suppose to be RosterState currentSpotRosterView;, somehow got merged with the previous variable and still compile and work fine)